src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/otel-adot-log-ingestion-serverless.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: OTEL log ingestion solution for ADOT lambda layer collector
+metaDescription: How to instrument your serverless applications on AWS Lambda with ADOT solution in New Relic.
+freshnessValidatedDate: never
+---
+You can now monitor AWS serverless application that sends OTEL log events data from CloudWatch Logs to the New Relic UI.
```

Comment:
On first use, always use the exploded version of the acronym (acronyms silo information and it make it harder for people to read and understand if they don't already know what the acronym is)

For example, OTEL -> OpenTelemetry

Later uses of acronyms are fine. 

We should always use OpenTelemetry and not OTEL also.

Reply:
Also, drop the "now" in "You can now monitor..."

Docs are meant to feel sort of timeless. That "now" is going to become outdated real fast

Reply:
Also, I think Lambda is always capitalized